pages/linux/cmus.md

```diff
@@ -0,0 +1,24 @@
+# cmus
+
+> Commandline C* Music Player.
+> Arrow keys to navogate, `<enter/return>` to select, numbers 1-8 switch between different views.
+
+- Open cmus from specified directory:
+
+`cmus {{path/to/directory}}`
+
+- Add file/directory to library:
+
+`:add {{/path/to/file_or_directory}}`
+
+- Pause/unpause current song:
+
+`c`
```

Comment:
For example: "Cease or continue playing the current song" (I know, "cease" is kind of a stretch, but it's technically correct :P)